mithril-explorer/src/constants.js

```diff
@@ -1 +1,15 @@
 export const aggregatorSearchParam = "aggregator";
+
+export const signedEntityType = {
+  MithrilStakeDistribution: "MithrilStakeDistribution",
+  CardanoStakeDistribution: "CardanoStakeDistribution",
+  CardanoImmutableFilesFull: "CardanoImmutableFilesFull",
+  CardanoTransactions: "CardanoTransactions",
+};
+
+export const defaultAggregatorCapabilities = {
```

Comment:
Do we really need to enforce the default aggregator capabilities?

Reply:
I wonder too, when I introduced this I had the idea of making all artifacts tabs conditional, this fallback would ensure that the "classic" tab would still show up.
But honestly I don't think that it have uses anymore.

Reply:
I removed it altogether.